dev/com.ibm.ws.app.manager.war/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -37,8 +37,9 @@
             <Option label="%earapp.addWebModuleClassPathTo.warLoader" value="war"/>
             <Option label="%earapp.addWebModuleClassPathTo.earLoader" value="ear"/>
         </AD>
+        <AD id="annotationScanLibrary" ibm:beta="true" name="%earapp.annotationScanLibrary.name" description="%earapp.annotationScanLibrary.desc" type="String" required="false"/>
```

Comment:
Should this attribute be like the one above it where there are `<Option/>`'s with labels so that you aren't stating the valid options in the description text of the attribute because that will be self-documented in the options?

Reply:
No because it is a comma separated list and not just one option.